packages/react-server-dom-webpack/src/ReactFlightDOMClientEdge.js

```diff
@@ -37,12 +37,12 @@ export function createServerReference<A: Iterable<any>, T>(
 }
 
 export type Options = {
-  moduleMap?: $NonMaybeType<SSRManifest>,
+  bundleConfig?: $NonMaybeType<BundleConfig>,
```

Comment:
"bundleConfig" inside an "options" is an unnecessary abstraction. The notion of a bundle config is an internal concept. The public API is just options. In other words, the Options object should just get another option added to it.

Reply:
I currently have chunkLoading as a required argument to the various `createFrom...` methods.

My thinking is that properly handling chunkLoading is so important it shouldn't be optional in the environments where it makes sense (Edge, Node). I considered defaulting it but there is no actually good default since a simple webpack config change would easily break it.

I could also infer it from webpack itself but that only works if you bundle react-server-dom-webpack which isn't necessarily what you want and it wouldn't help in Node's case